composer.json

```diff
@@ -27,6 +27,7 @@
         "symfony/mime": "^5.3.0",
         "symfony/translation": "^5.3.0",
         "symfony/yaml": "^5.3.0",
+        "symfony/runtime": "^5.3.0",
```

Comment:
Since this will be loaded on demand, by the developer in runtime configuration, we could consider putting this into `require-dev` section instead. Thoughts?